setup.cfg

```diff
@@ -31,7 +31,7 @@ install_requires =
     dill==0.3.6
     dnslib>=0.9.10
     dnspython>=1.16.0
-    plux>=1.3.1
+    plux>=1.7,<2.0
```

Comment:
I think this is a good opportunity to discuss upper version limits.
We explicitly removed pins like this in https://github.com/localstack/localstack/pull/10195.
We did this because there's no need for the pins for as long as the currently released version is compatible.
New releases will either be picked up proactively by us (in PRs like this, explicitly upgrading a single dependency), or they are picked up by the weekly upgrades where the breaking changes are identified and (a) fixed in the PR by adding a small fix, or (b) pinned and explicitly upgraded in a follow up PR (where the min version is updated and the upper version limit removed).
In this case, the library is maintained by us, meaning that we already explicitly know that `2.0` will contain breaking changes. However, if the person who creates the release does not proactively update it here, we most likely won't find out (because the automated updates won't dare to touch it 😛)...
This is why I am leaning towards not having these pins, but again, this is a great opportunity to discuss them. 😄